src/GovUk.Education.ExploreEducationStatistics.Data.Model.Tests/Repository/FootnoteRepositoryTests.cs

```diff
@@ -6,12 +6,2253 @@
+                    {
+                        Footnote = new Footnote
+                        {
+                            Content = "Applies to subject 1",
+                            Subjects = new List<SubjectFootnote>
+                            {
+                                new()
+                                {
+                                    Subject = subject1
+                                }
+                            }
+                        }
+                    },
+                    new()
+                    {
+                        Footnote = new Footnote
+                        {
+                            Content = "Applies to subject 1 filter 1",
+                            Filters = new List<FilterFootnote>
+                            {
+                                new()
+                                {
+                                    Filter = filter1
+                                }
+                            }
+                        }
+                    },
+                    new()
+                    {
+                        Footnote = new Footnote
+                        {
+                            Content = "Applies to subject 1 filter 2 group 1",
+                            FilterGroups = new List<FilterGroupFootnote>
+                            {
+                                new()
+                                {
+                                    FilterGroup = filter2.FilterGroups[0]
+                                }
+                            }
+                        }
+                    },
+                    new()
+                    {
+                        Footnote = new Footnote
+                        {
+                            Content = "Applies to subject 1 filter 3 group 1 item 1",
+                            FilterItems = new List<FilterItemFootnote>
+                            {
+                                new()
+                                {
+                                    FilterItem = filter3.FilterGroups[0].FilterItems[0]
+                                }
+                            }
+                        }
+                    },
+                    new()
+                    {
+                        Footnote = new Footnote
+                        {
+                            Content = "Applies to subject 1 indicator 1",
+                            Indicators = new List<IndicatorFootnote>
+                            {
+                                new()
+                                {
+                                    Indicator = indicatorGroup1.Indicators[0]
+                                }
+                            }
+                        }
+                    }
+                }
+            };
+
+            // Create another release which has no footnotes
+            var release2 = new Release();
+
+            var releaseSubjects = new List<ReleaseSubject>
```

Comment:
Is it worth adding a note here to say that we hope to be removing this ReleaseSubject link table soon, and so going forwards from that point this test case can be removed?

Just thinking that if we don't get round to that any time soon, and then when we do we run up against this explicit test case, we might second guess whether or not we should be removing the link table!

Worth mentioning at least that we *shouldn't* be seeing this kind of data under normal circumstances?

Reply:
It's the `ReleaseFootnote` table we are hoping to get rid of I think? See [EES-2979](https://dfedigital.atlassian.net/browse/EES-2979).

I'm not sure that we should get rid of this test even after we get rid of that link.

I've added this explanation into the test:

```
This test covers a case where a subject is shared by multiple releases. It makes sure that even though
a different release has footnotes for the subject, there's no way of retrieving the footnotes that
belong to another release by specifying the id, filter item id's and indicator id's of the subject.
```

Reply:
Yup thanks - that was just me getting confused with the different join tables - ta.